lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
+    get: function() {
+      delete this.engineVersions; 
+      return this.engineVersions = JSON.parse(
```

Comment:
Right, `delete` will remove the property, so you can then write to it because there’s no getter there. You’d only need `writable` to be `true` if the `delete` call weren’t there. But the `delete` call doesn’t work if `configurable` isn’t explicitly set to `true`.